src/layouts/entity.tsx

```diff
@@ -56,8 +60,29 @@ const EntityLayout = ({ data }: EntityLayoutProps) => {
   const showInsights = insights?.title && insights?.insights;
   const showReviews = reviews?.title && reviews?.reviews;
 
+  const { t } = useTranslation();
+  const count = Math.ceil(Math.random() * 2);
+
   return (
     <>
+      <Trans>Welcome to My House</Trans>
+      <div>
+        <Trans shouldUnescape={true}>
```

Comment:
Do we need to add shouldUnescape every time we have JSX in the string?

Reply:
No the prop isn't necessary, removed it.